cmd/lassie/fetch.go

```diff
@@ -277,12 +277,16 @@ func defaultFetchRun(
 		fmt.Fprintln(msgWriter)
 		return err
 	}
+	spid := stats.StorageProviderId
+	if spid == "" {
```

Comment:
We have a const for this -- https://github.com/filecoin-project/lassie/blob/main/pkg/types/types.go#L225